src/components/board/MeetTheBoard.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 const MeetTheBoard = () => {
   return (
     <div className="flex w-full flex-row justify-center text-3xl">
```

Comment:
You dont need the text-3xl in the outside `<div/> `tag since you already used it in the `<h1/>`